packages/react/src/dialog-description/DialogDescription.tsx

```diff
@@ -15,17 +16,9 @@ type DialogDescriptionProps = BoxProps<
 export const DialogDescription = forwardRef<
   HTMLParagraphElement,
```

Comment:
Vai, should it be a `HTMLParagraphElement` or `HTMLDivElement`?

Reply:
For drawer with the same structure, we used the HTMLDivElement, wasn't sure about it vai.